docs/vue/slides.md

```diff
@@ -53,7 +52,7 @@ We recommend importing the styles in the component in which Swiper is being used
 <script>
   import { defineComponent } from 'vue';
 
-  import 'swiper/css';
+  import 'swiper/swiper.css';
```

Comment:
What problems were you running into getting this to work with Vue? The Vue CLI definitely supports ESM (otherwise Ionic Vue would not work).

Reply:
When using an Ionic starter app, it's unable to resolve the import:
```
Module not found: Error: Can't resolve 'swiper/css' in 'C:\Users\moogl\Desktop\repos\_personal-sandboxes\ionic-vue\src\views'
```
Note that `swiper/css` is still the import shown in Swiper's usage docs: https://swiperjs.com/vue#usage

Reply:
Is there a repo I can try?

Reply:
Sure, here's the one I've been using: https://github.com/amandaejohnston/sandbox-ionic-vue/tree/swiper-migration-testing

Reply:
Issue here is your sample app is using Vue CLI 4 which ships with an older version of Webpack that does not support the `exports` feature Swiper 9 uses. Updating the Vue CLI 5 fixed the issue (might be easiest to copy an Ionic starter app into your test repo since that has all the dependencies)

Reply:
Huh, sure enough. Maybe I just created this test app before we upgraded to Vue CLI 5 and never updated it? I'll revert the import changes here.

Reply:
Alright, should be good to go now 👍